cli/mizu/logger.go

```diff
@@ -35,5 +34,6 @@ func InitLogger() {
 
 	logging.SetBackend(backend1Leveled, backend2Formatter)
 
+	Log.Debugf(" ===================== \n\n\n\n")
```

Comment:
Maybe \n twice is enough.. 
Also Im thinking if we need to know the origin of this (of mizu is running and user start another mizu in second terminal it can confuse us) - maybe the solution will be to add something to the log when we are doing this Init and it will be written in the logs and we will have context - even process ID or info about the process will help because it will be another one in the second command

Reply:
in the log file we have process id

Reply:
https://github.com/up9inc/mizu/blob/1ab45d4e2ce0b0825c165fc23e8947a9ff2f4b1d/cli/mizu/logger.go#L13

Reply:
Oh, my bad for not checking it..